python/taulabs/uavtalk.py

```diff
@@ -57,29 +57,46 @@ def process_stream(uavo_defs, use_walltime=False, gcs_timestamps=False):
 
     received = 0
 
-    packet_bytes = ''
+    buf = ''
+    buf_offset = 0
```

Comment:
Now that you have an offset, you can probably remove the comment above that says "This could be made considerably faster if it kept an offset."